exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -166,6 +178,18 @@ public HashCode submitUnknownTx() {
     });
   }
 
+  /**
+   * Increments the afterCommit counter so the number of times this method was invoked is stored
+   * in it.
+   */
+  @Override
+  public void afterCommit(BlockCommittedEvent event) {
```

Comment:
I'd consider moving it up to other methods defined in `Service`, and then have methods from `CryptocurrencyService`.